src/effects/native/balanceeffect.cpp

```diff
@@ -1,5 +1,9 @@
 #include "balanceeffect.h"
 
+namespace {
```

Comment:
This will be used for the low pass mono bypass.
I can remove it for now if you like.